CMakeLists.txt

```
@@ -68,9 +68,9 @@ set(OBJECTS_VERSION "1.3.11")
 set(OBJECTS_URL  "https://github.com/OpenRCT2/objects/releases/download/v${OBJECTS_VERSION}/objects.zip")
 set(OBJECTS_SHA1 "02ddf6c685a2da8bac98a57b34be95d63192443d")
 
-set(OPENSFX_VERSION "1.0.3")
+set(OPENSFX_VERSION "1.0.2")
```

Looks like an accidental revert.

```suggestion
set(OPENSFX_VERSION "1.0.3")
```